cmd/skywire-visor/commands/root.go

```diff
@@ -581,3 +446,144 @@ func checkHvIsRunning(addr string, retries int) bool {
 	}
```

Comment:
`isHvRunning` should be sufficient.

Reply:
this is not a function I have touched, I'm guessing @mrpalide made that one. I've changed the name and will see what i can do on the rest

Reply:
actually is older than me on skywire team :smile:, bdw isHvRunning is sufficient. :+1: